### ISSUE TITLE
Opponents move didn't show up, lost on time

### ISSUE THREAD

**Issue by wrzr123** [lichess-org/lichobile#1558]
**Describe the bug**
Some minutes ago, I played a standard 10+5 game in the lichess app against cakeboss22. My username is also wrzr123 in lichess. After my move 17, my opponent wasn't doing moves any more and his clock was running down to zero. After that nothing happened, no indication that I won the game and no rating points. I switched to my profile, looked at the game and saw that in fact I lost the game on time and my opponent did make a move after a couple of seconds. It didn't seem like I had a connection problem, everything was loading fine and quickly. 

I'm not sure if this is the right place to report problems like this, but I didn't find anything else in the app. It's not about requesting my rating points back, I just want to help to improve the software. 

**To Reproduce**
Probably hard to reproduce. Was the first time that happened to me, didn't play a game since then. 

**Expected behavior**
After my opponents move, I see it and my clock is running down, not his. 

**Screenshots**
Unfortunately I didn't make a Screenshot with my opponents clock down to 0 to prove what happened. 

**Smartphone (please complete the following information):**
 - Device: Huawei Mate 10 Light 
 - OS: Android 8.0.0
 - Browser: lichess App
 - Version: v7.7.2

**Comment by niklasf** [comment]
Thanks for reporting. This was due to https://lichess.org/forum/lichess-feedback/server-failure-this-morning, so nothing the app can do about it (except perhaps give an early warning after https://github.com/ornicar/lila/issues/8185 is implemented on the server).

**Comment by davinderc** [comment]
I had the same issue just now. I started a game, we played a few moves, but in my case i had to switch apps for a few seconds to show the train conductor my ticket. After coming back to lichess less than 5 seconds later i could see the opponent's clock ticking but they didn't move. When I asked, they said they were waiting for my move. I have a screenshot below. In the end i lost on time.

Samsung Galaxy S10+
Android 11
![Screenshot_20210401-140319_lichess](https://user-images.githubusercontent.com/19155750/113291937-0b0cdd00-92f4-11eb-890b-00bb93a4903f.jpg)
![Screenshot_20210401-140519_lichess](https://user-images.githubusercontent.com/19155750/113291939-0c3e0a00-92f4-11eb-93e0-7a8b537bc84f.jpg)
